tests/driver.c

```diff
@@ -266,7 +266,7 @@ int main(int argc, char* argv[]) {
   }
 
   ta_core_default_init(&ta_core);
-  ta_core_cli_init(&ta_core, argc, argv);
+  ta_core_cli_init(&ta_core, &argc, argv);
```

Comment:
Why?

Reply:
I have no idea why you would use expensive pointer to pass short integer.

Reply:
Or how about using a static integer in `config.c` to store the updated `argc`?

Reply:
Once argc/argv is parsed, we would not have to preserve it.

Reply:
But we would parse the CLI twice. The first time aims to find the configure file path and set the configures with configure file. The second time is using the CLI arguments to set TA configures.
Therefore the updated argc would be used in the second time.

Reply:
How about preserving in a static short int for argc?

Reply:
OK